Apps/WebClient/test/functional/e2e/cypress/integration/user/profile.js

```diff
@@ -1,17 +1,32 @@
 const { AuthMethod } = require("../../support/constants")
 
 describe('User Profile', () => {
-    beforeEach(() => {
+    const emailAddress = "healthgateway@mailinator" + Math.random().toString().substr(2, 5) + ".com"
+    before(() => {        
+        cy.server()
+        cy.fixture('AllDisabledConfig').as('config')
+        cy.route('GET', '/v1/api/configuration/', '@config')
         cy.login(Cypress.env('keycloak.username'), 
                  Cypress.env('keycloak.password'), 
                  AuthMethod.KeyCloak)
     })
 
-    it('Test1', () => {
-
+    it('Profile - Email Validation - email fields should be identical', () => {
```

Comment:
The test name is too long, you're in Profile so we don't need to replicate that and what we're doing is Email Validation.